### PR TITLE
Authorization levels on meetups

### LINKED ISSUE #322: Bug - Signals aren't working
## Description
Signals aren't being called hence groups for authorization aren't created after the migration of portal.  

## Mocks
After adding a meetup location or a community, the community/meetup location groups aren't created.
Create a meetup location, add a user to be a member. make the member an organiser. This member won't be having any permissions since the permissions groups aren't created.

### Update [Required]
- [ ]  Add apps.py
- [ ] Add tests 

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
2 hours

I'm working on this issue.

### PR DESCRIPTION
**Pull request by sa7i**
### Description
This pull request changes the authorization levels on meetups. Changes are made according to the issue #272
Fixes #322  and #272 
### Type of Change:
- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors).

### How Has This Been Tested?
image

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: systers_portal/community/apps.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class CommunityConfig(AppConfig):
+    name = 'community'
+
+    def ready(self):
+        import community.signals
+        assert community.signals
```
[the remark this answers is not in the document]
Comment (sa7i) [reply systers_portal/community/apps.py]: assert is added to remove `imported but unused` pep8 error. https://groups.google.com/forum/#!topic/comp.lang.python/nryFbibthpo

**Comment by tapasweni-pathak** [comment]
bump @sa7i.

**Comment by sa7i** [comment]
@tapasweni-pathak Could you please merge #352 before this? So I can make changes corresponding to the newest merged code in this PR.

**Comment by tapasweni-pathak** [comment]
Hey @yatna, got notification and was looking at it, I would suggest not closing this PR, this contains very important changes. I hope you and @systers/committers-portal had a sync about closing this pull request.

cc @systers/admins @systers/portal.

**Comment by tapasweni-pathak** [comment]
Most importantly if @sa7i is contributing than with her.

**Comment by sa7i** [comment]
@tapasweni-pathak @yatna Hi, I am reopening this PR to fix the conflicts and merge it.

**Comment by sa7i** [comment]
@yatna @tapasweni-pathak @abhi20161997 Please review this to merge it before the gsoc coding period starts.

**Pull request merged**